hedera-node/src/test/java/com/hedera/services/security/ops/SystemOpPoliciesTest.java

```diff
@@ -178,7 +150,18 @@ public void cryptoUpdateRecognizesAuthorized() throws InvalidProtocolBufferExcep
 	}
 
 	@Test
-	public void cryptoUpdateRecognizesUnnecessary() throws InvalidProtocolBufferException {
+	public void cryptoUpdateRecognizesUnnecessaryForSystem() throws InvalidProtocolBufferException {
+		// given:
+		var txn = civilianTxn()
+				.setCryptoUpdateAccount(CryptoUpdateTransactionBody
+						.newBuilder()
+						.setAccountIDToUpdate(account(75)));
+		// expect:
+		assertEquals(UNNECESSARY, subject.check(accessor(txn)));
+	}
```

Comment:
This should be `UNAUTHORIZED`! Civilian should not be able to update a system account. I think the idea of the issue #774 is that if account 2 or account 50 pays for this transaction to update account 75 (a system account), account 75 (target) doesn't need to sign this transaction, just 2 or 50 (payer) needs to sign.

Reply:
Yeah @qnswirlds is correct. The any account should be account 2 (TREASURE) or 50 (SYSTEM_ADMIN or previous MASTER).

Reply:
In fact this code does give the desired behavior; the point of #774 is that we **remove** the payer authorization requirement for system accounts. It may be helpful o read the [updated doc on privileged](https://github.com/hashgraph/hedera-services/blob/0774-M-NonPayerPrivsForCryptoUpdate/docs/privileged-transactions.md)

Reply:
Ah I see. Thanks for the clarification, @tinker-michaelj! I think I missed the point of **as long as 0.0.88 (or 0.0.75 as in the test) signs the transaction**, because the test could not show the whole flow of a transaction from the time it is received to the time it is handled. Then perhaps we should make it clearer in the `privileged-transactions.md` to:
- Remind about signing requirements on `CryptoUpdate` and `FileUpdate`/`FileAppend`. This should be a separate bullet/header
> `CryptoUpdate` and `FileUpdate`/`FileAppend` transactions must be signed with the target entity's key, or they will fail with status INVALID_SIGNATURE
- When explaining

> There are two kinds of privileges,
> 
> 1. Authorization - some transaction types, such as Freeze, require authorization to submit to the network. **All such transactions will be rejected with the status UNAUTHORIZED unless they are privileged.** (bold this part)
> 2. Waived signing requirements - privileged `CryptoUpdate` and `FileUpdate`/`FileAppend` transactions will be waived the signing requirement that was described in (reference to the separate bullet/header above)

- Also, add the scenario that you explained here as a note:
> Note that it is now possible to update 0.0.88 (a system account, for example) with any payer, **as long as 0.0.88 signs the transaction**.

Reply:
Added a clarifying example for readers used to pre-`0.10.0` behavior:

> (Note that before release `0.10.0`, a `CryptoUpdate` targeting _any_ system account required an  
authorized payer. Since `0.10.0` it has been possible to, for example, update `0.0.88` with 
`0.0.12345` as the payer, as long as the key for `0.0.88` signs the transaction.)

As I read the doc, it does emphasize the default requirement that target entities sign...we may want to refine later, or add a separate doc that covers all the signing requirements in one place. 👍